data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, to peer into space. "What a massive ringworld!", she exclaims.`
+			choice
+				`	"It's enormous. They actually stole it from the Quarg, who they rebelled against roughly six thousand years ago."`
+				`	"I'm used to it."`
+			`	"Awesome! This is going to be a great setting and certainly a source of inspiration for my next novel!", says Anna, her voice still in an enthusiastic and excited tone. "This must be where I'm supposed to register, right?"`
+			`	"Yes, this is the <planet>. Prepare your identity documents."`
+	on complete
+		conversation
+			`	Anna is still in awe as you touch down on the ringworld. As the two of you open the hatch, you see yet another Heliarch agent in front of you.`
+			`	"Greetings, humans. For you, what can I do today?", asks the Heliarch.`
+			`	"This is my friend Anna. I'm getting her registered with the Heliarchs today.", you answer.`
+			`	"To reach the registration office, this way you must go.", replies the Heliarch, pointing to a hallway.`
+			`	You and Anna walk down the hallway as directed, until you spot a registration office. Anna walks into the registration office while you sit down on a nearby bench. After 15 minutes, Anna walks out and approaches you. "They finally registered me. I'm off to the spaceport now to eat something. Meet me there so we can discuss our next move."`
+
+mission "Fangvin: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds, eating from a plate of vegetables, her laptop next to the plate of food. She sees you and motions for you to take a seat.`
+			`	You sit down next to Anna. "Finally, something I can eat that isn't a mix of vomit and soup!"`
+			choice
+				`	"How good is it?"`
+					goto "it's very good"
+				`	"What did you usually eat back in human space?"`
+			`	"A lot. As an author, all I had to eat were instant noodles and takeout food as I never could find time in my schedule to cook at home. I was working hard on Beyond the Hyperdrive at the time. This food is better than any instant noodle brand I could ever find on Zug."`
+			choice
+				`	"What's our plan now?"`
+				`	"What are we going to do next?"`
+					goto "i want to get my book published here"
+			label "it's very good"
+			`	"It's very good. Probably better than anything I've had back on Zug."`
+			choice
+				`	"So, what do you want to do now?"`
+			label "i want to get my book published here"
+			`	"I want to get my book published here."`
```

Comment:
This is a bit too short. There doesn't need to be some long paragraph to say this, it can be short, but this line feels kinda raw, for lack of a better word. Could just add another short sentence or two, just to not have the delivery feel dry.